conf/application.conf

```diff
@@ -18,25 +18,16 @@ include "backend.conf"
 
 appName=upscan-notify
 
-play.modules.enabled += "com.kenshoo.play.metrics.PlayModule"
-
 # An ApplicationLoader that uses Guice to bootstrap the application.
 play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
 
 # Primary entry point for all HTTP requests on Play applications
 play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
```

Comment:
Some more of these can be removed - `ApplicationLoader`, `RequestHandler`, `GraphiteMetricsModule`, `JsonErrorHandler`